### PR TITLE
cclient/http: Added content_type and accept to http_info_t

### PR DESCRIPTION
**Pull request by ifullgaz**
Added content_type and accept to http_info_t. Solves issue #508 

# Test Plan:
Usual test plan

### CODE REVIEW

File: cclient/http/tests/test_http.c [hunk 1]
```diff
@@ -110,6 +110,9 @@ const char* data =
 void test_http(void) {
   iota_client_service_t service = {{0}};
   service.http.host = "httpbin.org";
+  service.http.content_type = "application/x-www-form-urlencoded";
+  service.http.accept = "application/json";
+  service.http.host = "httpbin.org";
```
Comment (oopsmonk) [inline cclient/http/tests/test_http.c]: duplicated with line 112

**Review by oopsmonk — CHANGES_REQUESTED** [review]

File: cclient/http/tests/test_http.c (hunk 1 above)
Comment (ifullgaz) [reply cclient/http/tests/test_http.c]: Sorry...

**Review by th0br0 — COMMENTED** [review]
Could we provide constants for useful content-type/accept combinations?

e.g. `const char* http_kApplicationJson = "application/json";` + `extern char* http_kApplicationJson;`?

**Review by th0br0 — CHANGES_REQUESTED** [review]
see prev comment

**Review by oopsmonk — APPROVED** [review]

**Pull request merged**